docs/design/kubeadm-api.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+type MasterConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Data shared between phases. Used for defaulting
+	Master     Master     `json:"master"`
```

Comment:
This is not by far done; I'm trying to see how I can move stuff around to get rid of it